SPECS-EXTENDED/libwpg/libwpg.spec

```
@@ -0,0 +1,229 @@
+Summary:        A library for import of WordPerfect Graphics images
+Name:           libwpg
+Version:        0.3.3
+Release:        9%{?dist}
+License:        LGPL-2.0-or-later OR MPL-2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://libwpg.sourceforge.net/
+Source0:        http://download.sourceforge.net/libwpg/%{name}-%{version}.tar.xz
```

Let's use HTTPS whenever possible:
```suggestion
Source0:        https://download.sourceforge.net/libwpg/%{name}-%{version}.tar.xz
```